docs/configuration.rst

```diff
@@ -0,0 +1,141 @@
+
+``out_prefix :: str``: The output file base name. Defaults to ``forcing``.
+
+``verbose :: bool``: Whether to print output to ``stdout``.  Should be set using
+the ``-v`` flag for command line usage.  This can be set for scripting purposes,
+if desired. Set to ``1`` to print output; defaults to ``0``.
+
+``base_elev :: float``: Base offset of elevation (m).  Defaults to ``0``.
+
+``site_isoh :: float``: Isohyetal level offset for adjusting precipitation 
+inputs.  Default to ``1``.
+
+``base_isoh :: float``: Isoyetal base level for adjusting precipitation inputs. 
+Defaults to ``1``.
+
+``sw_prec_thresh :: float``: Minimum precipitation threshold to take into 
+account when simulating incoming shortwave radiation.  Defaults to ``0``.
+
+``mtclim_swe_corr :: bool``: Whether to activate MTCLIM's SWE correction 
+algorithm. Default to ``0``.
+
+``lw_cloud :: str``: Type of cloud correction to longwave radiation to apply. 
+Can be either ``DEFAULT`` or ``CLOUD_DEARDORFF``.  Defaults to 
+``CLOUD_DEARDORFF``.  Capitalization does not matter.
+
+``lw_type :: str``: Type of longwave radiation parameterization to apply. Can be
+one of the following: ``DEFAULT``, ``TVA``, ``ANDERSON``, ``BRUTSAERT``, 
+``SATTERLUND``, ``IDSO``, or ``PRATA``.  Defaults to ``PRATA``.  Capitalization 
+does not matter.
+
+``tdew_tol :: float``: Convergence criteria for the iterative calculation of 
+dewpoint temperature in MTCLIM.  Defaults to ``1e-3``.  
+
+``tmax_daylength_fraction :: float`` : Weight for calculation of time of maximum
+daily temperature.  Must be between ``0`` and ``1``.  Defaults to ``0.67``.
+
+``out_vars :: list`` : List of variables to write to output.  Should be a list 
+containing valid variables.  The list of valid variables is dependent on which 
+simulation method is used, as well as whether disaggregation is used. Defaults 
+to ``['temp', 'prec', 'shortwave', 'longwave', 'vapor_pressure', 'red_humid']``.
+
+
+in_vars section
+---------------
+The ``in_vars`` section is where you can specify which variables are in your 
+input data, and the corresponding symbols which MetSim will recognize. The 
+format of this section depends on the value given in the ``in_format`` entry in 
+the ``MetSim`` section of the configuration file.  See below for conventions for
+each input type.
+
+
+netcdf
+``````
+The ``in_vars`` section for NetCDF input acts as a mapping between the variable 
+names in the input dataset to the variable names expected by MetSim.  The format
+is given as ``netcdf_varname = metsim_varname``.  The minimum required variables 
+given have ``metsim_varname``s corresponding to ``t_min``, ``t_max``, and 
+``prec``; these variable names correspond to minimum daily temperature (Celcius), 
+maximum daily temperature (Celcius), and precipitation (mm).
+
+ascii
+`````
+The ``in_vars`` section for ASCII input acts similarly to the NetCDF input 
+format, except for one key point.  Variables should be given as a tautology: the 
```

Comment:
I've never used the word ``tautology`` without being forced to. Super impressed over here.